app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx

```diff
@@ -1,58 +1,70 @@
 'use client';
 
+import { type Participant } from '@prisma/client';
+import { useState } from 'react';
+import { trpc } from '~/app/_trpc/client';
 import { DataTable } from '~/components/DataTable/DataTable';
-import { useParticipants } from '../ParticipantsProvider';
-import { ParticipantColumns } from './Columns';
 import ParticipantModal from '../../participants/_components/ParticipantModal';
-import { useState } from 'react';
-import {
-  deleteParticipant,
-  deleteParticipants,
-} from '../../participants/_actions/actions';
-import { type Participant } from '@prisma/client';
+import { ParticipantColumns } from './Columns';
```

Comment:
Can you make sure to use import aliases for these and others across this PR?

Reply:
Thank you, I changed the import aliases